.eslintrc

```diff
@@ -31,7 +31,7 @@ rules:
 
   # Best Practices
   # http://eslint.org/docs/rules/#best-practices
-  no-multi-spaces: 2
+  no-multi-spaces: 0
```

Comment:
This was actually failing linting, hence I disabled it

Reply:
Mmh... how exactly was this rule failing the linting? 🤔 I’d check myself, but until the end of this week I only have access to an iPad and no real computer to do so. 🙇🏻‍♂️

Reply:
@arthurschreiber 

```
/Users/igor.savin/WebstormProjects/tedious-f2/src/bulk-load.js
  15:29  error  Multiple spaces found before '// introduced ...'  no-multi-spaces
  16:29  error  Multiple spaces found before '// introduced ...'  no-multi-spaces
  18:29  error  Multiple spaces found before '// introduced ...'  no-multi-spaces
  19:29  error  Multiple spaces found before '// introduced ...'  no-multi-spaces
  20:29  error  Multiple spaces found before '// introduced ...'  no-multi-spaces

/Users/igor.savin/WebstormProjects/tedious-f2/src/connection.js
   765:60  error  Multiple spaces found before '// EOM pseudo ...'  no-multi-spaces
   857:60  error  Multiple spaces found before '// release old...'  no-multi-spaces
  1475:60  error  Multiple spaces found before '// Request.pau...'  no-multi-spaces
  1759:60  error  Multiple spaces found before '// request tim...'  no-multi-spaces

/Users/igor.savin/WebstormProjects/tedious-f2/src/token/stream-parser.js
  38:60  error  Multiple spaces found before '// generate en...'  no-multi-spaces

/Users/igor.savin/WebstormProjects/tedious-f2/test/unit/login7-payload-test.js
  100:53  error  Multiple spaces found before '// Length'      no-multi-spaces
  101:53  error  Multiple spaces found before '// Variable'    no-multi-spaces
  107:53  error  Multiple spaces found before '// Reserved'    no-multi-spaces
  112:47  error  Multiple spaces found before '// NTLM'        no-multi-spaces
  115:53  error  Multiple spaces found before '// cbSSPILong'  no-multi-spaces
```